docs/visual-basic/programming-guide/concepts/linq/adding-elements-attributes-and-nodes-to-an-xml-tree.md

```diff
@@ -4,76 +4,73 @@ ms.date: 07/20/2015
 ms.assetid: e243e694-c987-43aa-8b22-1e33dace582c
 ---
 # Adding Elements, Attributes, and Nodes to an XML Tree (Visual Basic)
-You can add content (elements, attributes, comments, processing instructions, text, and CDATA) to an existing XML tree.  
-  
-## Methods for Adding Content  
- The following methods add child content to an <xref:System.Xml.Linq.XElement> or an <xref:System.Xml.Linq.XDocument>:  
-  
-|Method|Description|  
-|------------|-----------------|  
-|<xref:System.Xml.Linq.XContainer.Add%2A>|Adds content at the end of the child content of the <xref:System.Xml.Linq.XContainer>.|  
-|<xref:System.Xml.Linq.XContainer.AddFirst%2A>|Adds content at the beginning of the child content of the <xref:System.Xml.Linq.XContainer>.|  
-  
- The following methods add content as sibling nodes of an <xref:System.Xml.Linq.XNode>. The most common node to which you add sibling content is <xref:System.Xml.Linq.XElement>, although you can add valid sibling content to other types of nodes such as <xref:System.Xml.Linq.XText> or <xref:System.Xml.Linq.XComment>.  
-  
-|Method|Description|  
-|------------|-----------------|  
-|<xref:System.Xml.Linq.XNode.AddAfterSelf%2A>|Adds content after the <xref:System.Xml.Linq.XNode>.|  
-|<xref:System.Xml.Linq.XNode.AddBeforeSelf%2A>|Adds content before the <xref:System.Xml.Linq.XNode>.|  
-  
-## Example  
-  
-### Description  
- The following example creates two XML trees, and then modifies one of the trees.  
-  
-### Code  
-  
-```vb  
-Dim srcTree As XElement = _  
```

Comment:
It could be necessary in other contexts. But in this case, they're redundant. Even Visual Studio removed them automatically without asking when I copied the code.